src/Quartz.Extensions.DependencyInjection/MicrosoftDependencyInjectionJobFactory.cs

```diff
@@ -10,19 +11,19 @@ namespace Quartz
     internal sealed class MicrosoftDependencyInjectionJobFactory : PropertySettingJobFactory
     {
         private readonly IServiceProvider serviceProvider;
-        private readonly JobFactoryOptions options;
+        private readonly QuartzOptions options;
```

Comment:
If `MicrosoftDependencyInjectionJobFactory` is long-lived, maybe it would be better to have a reference to `IOptions<QuartzOptions>` here and call `options.Value` in `InstantiateJob`. (which would enable options reload support).

Reply:
Of course, changed.